### PR TITLE
Add Feature: Use last photo

### PR DESCRIPTION
**Pull request by Seachaos**
support android / ios use user last photo

### CODE REVIEW

**Comment by marcshilling** [comment]
@Seachaos sorry for the delay on reviewing this. This would be a cool feature to add. Can you resolve the conflicts and rebase on top of develop?

**Comment by Seachaos** [comment]
OK I will fix it.
Thanks.

**Comment by Seachaos** [comment]
It's has been resolve : )

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 1]
```diff
@@ -335,6 +350,131 @@ public void launchImageLibrary(final ReadableMap options, final Callback callbac
...
+      responseHelper.cleanResponse();
+      this.hasImagePathToCallback( imageLocation );
+    } else {
+      responseHelper.putString("error", "Cannot find last photo");
+      responseHelper.invokeResponse(callback);
+    }
+  }
+
+  private void hasImagePathToCallback(String realPath ) {
+    int currentRotation = 0;
```
Comment (rusfearuth) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java]: Could you use MediaUtils.readExifInterface? You've just duplicated code.

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 2]
```diff
@@ -335,6 +350,131 @@ public void launchImageLibrary(final ReadableMap options, final Callback callbac
...
+      responseHelper.putBoolean("isVertical", isVertical);
+    } catch (IOException e) {
+      e.printStackTrace();
+      responseHelper.putString("error", e.getMessage());
+      responseHelper.invokeResponse(callback);
+      this.callback = null;
+      return;
+    }
+
+    BitmapFactory.Options options = new BitmapFactory.Options();
```
Comment (rusfearuth) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java]: Look at onActivityResult method. You've duplicated code again.

**Comment by IcebergRage** [comment]
2 years later, it's still a good idea @Seachaos  :)

**Pull request closed without merging**